src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,141 @@
+package duke;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.DukeIDKException;
+import duke.exceptions.DukeInvalidDesException;
+import duke.exceptions.DukeMissingDesException;
+import duke.handler.Parser;
+import duke.handler.Queries;
+import duke.tasks.*;
+import duke.tasks.Event;
+
+import java.awt.*;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Duke {
+    private boolean exit = false;
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    private Duke(String storagePath) {
+        storage = new Storage(storagePath);
+        tasks = storage.load();
+        ui = new Ui();
+    }
+
+    public static void main(String[] args) throws IOException {
+        new Duke("data/tasks.txt").run();
```

Comment:
You may want to use relative path

Reply:
Hi my understanding was that this already is a relative path.
What do you mean by use a "relative path"? could you suggest how this is done? Thank you.